tools/SendBlobs/SetupCli.cs

```diff
@@ -31,7 +31,7 @@ public static void SetupExecute(CommandLineApplication app)
 
         app.OnExecuteAsync(async cancellationToken =>
         {
-            string? rpcUrl = rpcUrlOption.Value();
+            string rpcUrl = rpcUrlOption.Value()!;
```

Comment:
@flcl42 As I'm not familiar with the tool, I just added null-forgiving operators for CLI options to suppress warnings. A better solution is to either provide some default values or make those options mandatory with meaningful error messages. We can go this way for now until you decide what to do. Otherwise, feel free to commit changes to this branch.

Reply:
Could you add IsRequired to first several options also `CommandOption rpcUrlOption = app.Option("--rpcurl <rpcUrl>", "Url of the Json RPC.", CommandOptionType.SingleValue, (c) => c.IsRequired());`?

Reply:
@flcl42 Done